modules/administration_manual/pages/appliance/Clamav.adoc

```diff
@@ -46,12 +46,11 @@ refer to
 the ClamAV documentation <configure_clamav_antivirus_scanner_label> for
 instructions on how to do that.
 
-___________________________________________________________________________________________________________________________________________________________________________________________________________
+""
```

Comment:
I believe this is an admonition that didn't get converted properly.

Reply:
It renders now like this and that is imho looking ok.
![image](https://user-images.githubusercontent.com/3321281/49359837-9dc2f000-f6d7-11e8-91a0-9de0137a9d79.png)

Reply:
But that text reads as an admonition, not a quote.